### PR TITLE
[FRONTEND DEVELOPMENT: HTML,CSS,JS]: HTML LIST

### LINKED ISSUE #3245: [Frontend with HTML,CSS,JS]: HTML LIST
### Description

What is List ?
Types Of List
1)Ordered List
2)Unordered List

### Domain

Frontend Dev HTML/CSS/JS

### Type of Contribution

Documentation

### Code of Conduct

- [X] I follow [Contributing Guidelines](https://github.com/girlscript/winter-of-contributing/blob/main/.github/CONTRIBUTING.md) & [Code of conduct](https://github.com/girlscript/winter-of-contributing/blob/main/.github/CODE_OF_CONDUCT.md) of this project.

### PR DESCRIPTION
**Pull request by samipak458**
<hr>

## Description 📜
Documentation on Html list and types of list.
Fixes #3245

<hr>

## Type of change 📝

- [ ] Au#dio (Should be in mp3 format Includes speech clarity, Concise ,Low distortion)
- [ ] Vi#deo (Animations, screen-recordings, presentations and regular explanatory films are all possibilties etc)
- [X] Documentation (Content Creation in the form of codes or tutorials)
- [ ] Other (If you choose other, Please mention changes below) 

<hr>

## Domain of Contribution 📊

- [ ] Android Dev #(Flutter)
- [ ] Android Dev #(Java)
- [ ] Android Dev #(Kotlin)
- [ ] Backend Dev #(Java)
- [ ] Backend Dev #(.NET)
- [ ] Backend Dev #(PHP)
- [ ] Backend Dev #(Python)
- [ ] C/#CPP
- [ ] Competitive #Programming
- [ ] Cyber #Security
- [ ] DS#A
- [ ] Data#base
- [ ] Datascience with #Python
- [ ] Datascience with #R
- [X] Frontend Dev HTML/CSS/JS
- [ ] Frontend Dev #React/Angular/Vue
- [ ] Go#lang
- [ ] Interview #Prep
- [ ] Java_#Domain
- [ ] Java#Script
- [ ] ME#RN
- [ ] Machine #Learning
- [ ] Open #Source
- [ ] Python_#Domain
- [ ] Ru#st
- [ ] Statis#tics
- [ ] UI/#UX

<hr>

## Checklist ✅

- [X] I follow [Contributing Guidelines](https://github.com/girlscript/winter-of-contributing/blob/main/.github/CONTRIBUTING.md) & [Code of conduct](https://github.com/girlscript/winter-of-contributing/blob/main/.github/CODE_OF_CONDUCT.md) of this project.
- [X] I have performed a self-review of my own code or work.
- [X] I have commented my code, particularly in hard-to-understand areas. 
- [X] My changes generates no new warnings.
- [X] I'm GWOC'21 contributor

<hr>

## Screenshots / Gif (Optional) 📸

<hr>

### CODE REVIEW

**Comment by Anup0099** [comment]
@samipak458 Please assign this issue to me

File: Web_Development/FrontEnd/HTML_LIST.txt [hunk 1]
```diff
@@ -0,0 +1,38 @@
+
+
```
Comment (praveenscience) [inline Web_Development/FrontEnd/HTML_LIST.txt:2]: No need of blank spaces.

File: Web_Development/FrontEnd/HTML_LIST.txt [hunk 2]
```diff
@@ -0,0 +1,38 @@
+
+
+Question: What is List in HTML ?
```
Comment (praveenscience) [inline Web_Development/FrontEnd/HTML_LIST.txt:3]: Add a heading.

Comment (praveenscience) [reply Web_Development/FrontEnd/HTML_LIST.txt:3]: Remove space before `?`.

File: Web_Development/FrontEnd/HTML_LIST.txt [hunk 3]
```diff
@@ -0,0 +1,38 @@
+
+
+Question: What is List in HTML ?
+
+Answer: A number of items or information noted in order form or in bullet points is called "List". 
+
```
Comment (praveenscience) [inline Web_Development/FrontEnd/HTML_LIST.txt:6]: No need of extra line space.

File: Web_Development/FrontEnd/HTML_LIST.txt [hunk 4]
```diff
@@ -0,0 +1,38 @@
...
+Question: What is List in HTML ?
+
+Answer: A number of items or information noted in order form or in bullet points is called "List". 
+
+
+Question: Types of List ?
+Answer: There are two types of list:
+1) Ordered List
+2) Unordered List
```
Comment (praveenscience) [inline Web_Development/FrontEnd/HTML_LIST.txt:11]: Make them proper paragraph and not text.

File: Web_Development/FrontEnd/HTML_LIST.txt [hunk 5]
```diff
@@ -0,0 +1,38 @@
...
+
+
```
Comment (praveenscience) [inline Web_Development/FrontEnd/HTML_LIST.txt:13]: Remove empty space.

File: Web_Development/FrontEnd/HTML_LIST.txt [hunk 6]
```diff
@@ -0,0 +1,38 @@
...
+Ordered List:
```
Comment (praveenscience) [inline Web_Development/FrontEnd/HTML_LIST.txt:14]: Make it heading.

Comment (praveenscience) [reply Web_Development/FrontEnd/HTML_LIST.txt:14]: Leave a space after heading.

File: Web_Development/FrontEnd/HTML_LIST.txt [hunk 7]
```diff
@@ -0,0 +1,38 @@
...
+In ordered list orders matter. An ordered list starts with the <ol> tag. Each list item starts with the <li> tag and end with </li>. After all items added in list then we end ordered list with </ol>.
```
Comment (praveenscience) [inline Web_Development/FrontEnd/HTML_LIST.txt:15]: Make all tags inline codes.

File: Web_Development/FrontEnd/HTML_LIST.txt [hunk 8]
```diff
@@ -0,0 +1,38 @@
...
+
+Syntax:
+
+<ol>
+<li>Apple</li>
+<li>Mango</li>
+<li>Strawberry</li>
+</ol>
```
Comment (praveenscience) [inline Web_Development/FrontEnd/HTML_LIST.txt:23]: Code fence it with syntax highlighting for HTML.

File: Web_Development/FrontEnd/HTML_LIST.txt [hunk 9]
```diff
@@ -0,0 +1,38 @@
...
+Question: Types of List ?
+Answer: There are two types of list:
+1) Ordered List
+2) Unordered List
+
+
+Ordered List:
+In ordered list orders matter. An ordered list starts with the <ol> tag. Each list item starts with the <li> tag and end with </li>. After all items added in list then we end ordered list with </ol>.
+
+Syntax:
```
Comment (praveenscience) [inline Web_Development/FrontEnd/HTML_LIST.txt:17]: Make it a heading.

File: Web_Development/FrontEnd/HTML_LIST.txt [hunk 10]
```diff
@@ -0,0 +1,38 @@
...
+
+<ul>
+<li>2 pencils</li>
+<li>1 register</li>
+<li>5 pens</li>
+</ul>
+
+
```
Comment (praveenscience) [inline Web_Development/FrontEnd/HTML_LIST.txt:37]: Remove empty space.

File: Web_Development/FrontEnd/HTML_LIST.txt [hunk 11]
```diff
@@ -0,0 +1,38 @@
...
+<ol>
+<li>Apple</li>
+<li>Mango</li>
+<li>Strawberry</li>
+</ol>
+
+
+Unordered List:
+In unordered list order doesn't matter. An unordered list starts with the <ul> tag. Each list item starts with the <li> tag and end with </li>. After all items added in list then we end ordered list with </ul>.
```
Comment (praveenscience) [inline Web_Development/FrontEnd/HTML_LIST.txt:27]: Make all tags inline codes.

File: Web_Development/FrontEnd/HTML_LIST.txt [hunk 12]
```diff
@@ -0,0 +1,38 @@
...
+Syntax:
+
+<ol>
+<li>Apple</li>
+<li>Mango</li>
+<li>Strawberry</li>
+</ol>
+
+
+Unordered List:
```
Comment (praveenscience) [inline Web_Development/FrontEnd/HTML_LIST.txt:26]: Make it heading.

Comment (praveenscience) [reply Web_Development/FrontEnd/HTML_LIST.txt:26]: Leave space after heading.

File: Web_Development/FrontEnd/HTML_LIST.txt [hunk 13]
```diff
@@ -0,0 +1,38 @@
...
+
+Syntax:
+
+<ol>
+<li>Apple</li>
+<li>Mango</li>
+<li>Strawberry</li>
+</ol>
+
+
```
Comment (praveenscience) [inline Web_Development/FrontEnd/HTML_LIST.txt:25]: Remove blank lines.

File: Web_Development/FrontEnd/HTML_LIST.txt [hunk 14]
```diff
@@ -0,0 +1,38 @@
...
+Syntax:
+
+<ul>
+<li>2 pencils</li>
+<li>1 register</li>
+<li>5 pens</li>
+</ul>
+
+
+Reference Link: https://www.w3schools.com/html/html_lists.asp
```
Comment (praveenscience) [inline Web_Development/FrontEnd/HTML_LIST.txt:38]: Wrong place to refer. Never go for W3Schools.

**Review by praveenscience — CHANGES_REQUESTED** [review]
Please fix the changes.

File: Web_Development/FrontEnd/HTML_LIST.txt [hunk 15]
```diff
@@ -0,0 +1,45 @@
...
+<title>Html Unordered List</title>
+</head>
+<body>
+<ul>
+<li>2 pencils</li>
+<li>1 register</li>
+<li>5 pens</li>
+</ul>
+</body>
+</html>
```
Comment (praveenscience) [inline Web_Development/FrontEnd/HTML_LIST.txt:45]: Code fence it with syntax highlighting for HTML.

**Review by praveenscience — CHANGES_REQUESTED** [review]
Just two.

**Review by praveenscience — APPROVED** [review]
👍

File: Web_Development/FrontEnd/HTML_LIST.txt [hunk 16]
```diff
@@ -0,0 +1,48 @@
+##HTML LIST AND TYPES OF LIST
+
+**Question:** What is List in HTML?
+**Answer:** A number of items or information noted in order form or in bullet points is called "List". 
+
+**Question:** Types of List?
+**Answer:** There are two types of list; ordered List and unordered List.
```
Comment (kashishahuja2002) [inline Web_Development/FrontEnd/HTML_LIST.txt:7]: There is a dl also, don't just add incomplete information, add dl also.

**Review by kashishahuja2002 — CHANGES_REQUESTED** [review]
Add output ss for ordered and unordered lists.
Also add content for dl.

**Review by kashishahuja2002 — APPROVED** [review]

File: Web_Development/FrontEnd/HTML_LIST.md [hunk 17]
```diff
@@ -0,0 +1,82 @@
+##HTML LIST AND TYPES OF LIST
```
Comment (kashishahuja2002) [inline Web_Development/FrontEnd/HTML_LIST.md:1]: The hashtags should not be visible in the preview. Change that.

File: Web_Development/FrontEnd/HTML_LIST.md [hunk 18]
```diff
@@ -0,0 +1,82 @@
+##HTML LIST AND TYPES OF LIST
+
+**Question:** What is List in HTML?
+**Answer:** A number of items or information noted in order form or in bullet points is called "List".
```
Comment (kashishahuja2002) [inline Web_Development/FrontEnd/HTML_LIST.md:4]: Add an empty line before this line.

File: Web_Development/FrontEnd/HTML_LIST.md [hunk 19]
```diff
@@ -0,0 +1,82 @@
+##HTML LIST AND TYPES OF LIST
+
+**Question:** What is List in HTML?
+**Answer:** A number of items or information noted in order form or in bullet points is called "List". 
+
+**Question:** Types of List?
+**Answer:** There are three types of list; ordered list, unordered list and description list.
```
Comment (kashishahuja2002) [inline Web_Development/FrontEnd/HTML_LIST.md:7]: Add an empty line before this line.

File: Web_Development/FrontEnd/HTML_LIST.md [hunk 20]
```diff
@@ -0,0 +1,82 @@
+##HTML LIST AND TYPES OF LIST
+
+**Question:** What is List in HTML?
+**Answer:** A number of items or information noted in order form or in bullet points is called "List". 
+
+**Question:** Types of List?
+**Answer:** There are three types of list; ordered list, unordered list and description list.
+
+###Ordered List
```
Comment (kashishahuja2002) [inline Web_Development/FrontEnd/HTML_LIST.md:9]: The hashtags should not be visible in the preview. Change that.

File: Web_Development/FrontEnd/HTML_LIST.md [hunk 21]
```diff
@@ -0,0 +1,82 @@
...
+<ol>
+<li>Apple</li>
+<li>Mango</li>
+<li>Strawberry</li>
+</ol>
+</body>
+</html>
+```
+**Output:**
+![ ](https://drive.google.com/file/d/1_QYCFdOvE4mLsNa8ut4dhqRf1vyVnoi8/view?usp=sharing)
```
Comment (kashishahuja2002) [inline Web_Development/FrontEnd/HTML_LIST.md:31]: The image is not visible in the preview. Check that.

File: Web_Development/FrontEnd/HTML_LIST.md [hunk 22]
```diff
@@ -0,0 +1,82 @@
...
+
+###Unordered List
```
Comment (kashishahuja2002) [inline Web_Development/FrontEnd/HTML_LIST.md:33]: The hashtags should not be visible in the preview. Change that.

File: Web_Development/FrontEnd/HTML_LIST.md [hunk 23]
```diff
@@ -0,0 +1,82 @@
...
+<ul>
+<li>2 pencils</li>
+<li>1 register</li>
+<li>5 pens</li>
+</ul>
+</body>
+</html>
+```
+**Output:**
+![ ](https://drive.google.com/file/d/1CBDvZN4eWQzI2ClpSC5xs9ETSQwq1axL/view?usp=sharing)
```
Comment (kashishahuja2002) [inline Web_Development/FrontEnd/HTML_LIST.md:55]: The image is not visible in the preview. Check that.

File: Web_Development/FrontEnd/HTML_LIST.md [hunk 24]
```diff
@@ -0,0 +1,82 @@
...
+
+###Description List
```
Comment (kashishahuja2002) [inline Web_Development/FrontEnd/HTML_LIST.md:57]: The hashtags should not be visible in the preview. Change that.

File: Web_Development/FrontEnd/HTML_LIST.md [hunk 25]
```diff
@@ -0,0 +1,82 @@
...
+<dt>Banana:</dt>  
+<dd>It may improve Digestive Health.</dd>  
+<dt>Apricot:</dt>  
+<dd>It may protect your liver.</dd>  
+</dl>  
+</body>
+</html>
+```
+**Output:**
+![ ](https://drive.google.com/file/d/1gdigEJk6sra5B55KGSrvwvYKpfo-jOD9/view)
```
Comment (kashishahuja2002) [inline Web_Development/FrontEnd/HTML_LIST.md:82]: The image is not visible in the preview. Check that.

**Review by kashishahuja2002 — CHANGES_REQUESTED** [review]
Remove the .txt file.

**Review by kashishahuja2002 — CHANGES_REQUESTED** [review]
Don't include tabs and space below in the output image

**Review by kashishahuja2002 — APPROVED** [review]
Good to go. @praveenscience

**Pull request merged**